projects/angular/src/modal/side-panel.ts

```diff
@@ -158,15 +183,31 @@ export class ClrSidePanel implements OnInit, OnDestroy, OnChanges {
     }
   }
 
-  private displaySideBySide() {
-    this.hostElement.classList.add('clr-side-panel-pinned-' + this.size);
+  private updateModalState() {
+    if (!this.modal) {
+      return;
+    }
+    if (this.pinned) {
+      this.modal.stopClose = true;
+      this.sideBySideMargins();
+    } else {
+      this.modal.stopClose = this.originalStopClose;
+      this.cleanupMargins();
+    }
   }
 
-  private displayOverlapping() {
-    this.hostElement.classList.forEach(className => {
-      if (className.startsWith('clr-side-panel-pinned-')) {
-        this.hostElement.classList.remove(className);
-      }
+  private cleanupMargins() {
```

Comment:
Just a suggestion as it's more straightforward for what it's doing and this classes are adding padding currently.
```suggestion
  private cleanupSizingClasses() {
```

Reply:
Will proceed in style with the decision taken on the above question.

Reply:
Done -> cleanupPinnedClassses